libs/auth-api-lib/src/lib/user-identities/user-identities.service.ts

```diff
@@ -241,6 +241,8 @@ export class UserIdentitiesService {
       )
     })
 
-    return this.claimModel.findAll({ where: { subjectId: subjectId } })
+    return this.claimModel.findAll({
+      where: { subjectId: subjectId, useMaster: true },
```

Comment:
`useMaster: true` should be outside the `where` object.

Reply:
Yikes. Fixed.